components/validator/config.go

```diff
@@ -57,6 +60,9 @@ type CLIConfig struct {
 	// L1EthRpc is the HTTP provider URL for L1.
 	L1EthRpc string
 
+	// L1EthWsRpc is the Websocket provider URL for L1.
+	L1EthWsRpc string
```

Comment:
I think we don't need additional `L1EthWsRpc`, we just change `L1EthRpc` to websocker URL as node.